File: course2/sem3/hw1/task2/task2/task2/CustomRandom.cs
using System;

namespace Task2
{
    /// <summary>
    /// Custom random class
    /// </summary>
    public sealed class CustomRandom : ICustomRandom
    {
        int ICustomRandom.Random()
        {
            Random rand = new Random();
            return rand.Next(100);
        }
    }
}

File: course2/sem4/hw6/tasks/CustomRandom.fs
namespace Network

module CustomRandom = 

    /// <summary>
    /// Interface for custom random
    /// </summary>
    type ICustomRandom = 
        abstract member Random: unit -> int
    
    /// <summary>
    /// Custom rando class
    /// </summary>
    type CustomRandom() = 
        interface ICustomRandom with
            member this.Random() = System.Random().Next(100)
